example/gardener-local/gardenlet/values-kind-ha-sz.yaml

```diff
@@ -0,0 +1,33 @@
+global:
```

Comment:
Instead of duplicating the content, can't you just patch the `seedConfig.spec.highAvailability.failureTolerance` field depending on the set environment variable? Then we just have one `values-kind-ha.yaml` file and don't repeat ourselves unnecessarily...

Reply:
Fixed